data/src/main/java/org/apache/iceberg/data/DeleteLoader.java

```diff
@@ -42,4 +42,14 @@ public interface DeleteLoader {
    * @return a position delete index for the provided data file path
    */
   PositionDeleteIndex loadPositionDeletes(Iterable<DeleteFile> deleteFiles, CharSequence filePath);
+
+  /**
+   * Loads the content of the deletion vector into a position index.
+   *
+   * @param dv a deletion vector
+   * @return a position delete index for the provided deletion vector
+   */
+  default PositionDeleteIndex loadDV(DeleteFile dv) {
```

Comment:
I think I prefer #11481. This just feels like an unnecessary differentiation of an implementation that we can hide under the existing interface.

Reply:
Let's do #11481 then!

Reply:
I agree with @danielcweeks : I also prefer #11481 as an easier impl.